src/main/java/edu/wpi/first/smartdashboard/properties/StringProperty.java

```diff
@@ -13,6 +13,19 @@ public StringProperty(PropertyHolder element, String name, String defaultValue)
     super(String.class, element, name, defaultValue);
   }
 
+  @Override
+  public String getSaveValue() {
+    // In XML, the characters "&" and "<" are not valid in values by themselves
+    // Therefore here we replace these invalid characters by their escape sequences.
+    // Retrieve the save value from the superclass
+    String saveValue = super.getSaveValue();
+    // Replace &
+    saveValue = saveValue.replace("&", "&amp;");
+    // Replace <
+    saveValue = saveValue.replace("<", "&lt;");
+    return saveValue;
```

Comment:
You should also escape `>`, `'`, and `"`.

Reply:
Yes sorry for the delay. I have just added a commit for this, but since I used a tag instead of a branch for this PR, I have to resubmit it to update it. Sorry about that.

Reply:
I've reopened this at https://github.com/wpilibsuite/SmartDashboard/pull/124.